include/libopencm3/stm32/common/spi_common_v12.h

```diff
@@ -0,0 +1,390 @@
+/** @addtogroup spi_defines
+ *
+ * @author @htmlonly &copy; @endhtmlonly 2009 Uwe Hermann <uwe@hermann-uwe.de>
```

Comment:
Did not change this because I copied the file, but the contents is not written by me.